src/use-stepper.ts

```diff
@@ -196,6 +197,8 @@ function useStepper({
     const { onClick, ...otherIncrementProps } = incrementProps;
     return {
       ...otherIncrementProps,
+      'aria-hidden': true,
```

Comment:
I'm not sure `aria-hidden` is the right thing to do here. I don't see it mentioned in the spinbutton spec, but I would think it would need `aria-label` if not also `aria-controls`?

If we do consider `aria-label`, that brings up a bigger question around labels in general. Currently, they're left to the consumer entirely. If we get in the label business here, we'll need to require an `id` and introduce `getLabelProps()`, etc. That could be follow-on work, though.

Reply:
Agreed, let's add `tabIndex: -1` to them for sure and we can leave `aria-hidden` for now but may need to revise later. And yeah, we'll need to add key up/down - not sure it needs to happen in this PR or if we can do it after, though.